### PR TITLE
feat: style community server pages

### PR DESCRIPTION
**Pull request by JasperVandenberghen**
Added custom NDE styling for CSS signup pages.

`start:watch` and go to http://localhost:3000/

### CODE REVIEW

File: packages/solid-crs-pods/assets/register.html.ejs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>Registreren</h1>
+<form method="post" id="mainForm">
+
+  <% if (locals.message) { %>
+    <p class="error">Error: <%= message %></p>
+  <% } %>
+
+  <%- include('./register-partial.html.ejs', { allowRoot: false }) %>
+
+  <p class="actions"><button type="submit" name="submit">REGISTREER</button></p>
```
Comment (JasperVandenberghen) [inline packages/solid-crs-pods/assets/register.html.ejs:10]: @lem-onade I remember a comment in Figma saying the button should be in uppercase but looking at the other screens all the buttons seem to be lowercase with a capital letter?
```suggestion
  <p class="actions"><button type="submit" name="submit">Registreer</button></p>
```
```suggestion
  <p class="actions"><button type="submit" name="submit">REGISTREER</button></p>
```

**Review by woutermont — CHANGES_REQUESTED** [review]
Some thoughts:
- Let's keep the templates in the `template` folder, since they are not assets.
- Let's keep the structure of that template folder to mirror that of the CSS itself.
- Can you try to use `@ndesc:...` to refer to template files in the config files?
- Can you try to use `files-ndesc:config/...` to refer to config files in the import of other config files?

I only just realised the last two should in fact work.

**Review by lem-onade — CHANGES_REQUESTED** [review]
currently still getting errors on /idp/register:

```Error: ENOENT: no such file or directory, open '/Users/stijn/Documents/digita/nde-erfgoedinstellingen/packages/solid-crs-pods/assets/main.html.ejs'```

i think the assets -> templates file move still needs to be done in the config files

**Review by lem-onade — CHANGES_REQUESTED** [review]
- [x] The image at the top is different/larger than the one in Figma. (You can export as png, jpg from Figma) Image should be around 300x125, is now 342×209 px

- [ ] Spacing between some parts of the form is different from Figma (above 'Uw WebID', 'Uw Pod', the submit button). Also some missing margin at the bottom of the page. (white card sticks to bottom of page) 

- [x] Input fields have a total height of 40px in Figma, included is vertical padding of +- 5px for the text. Horizontal padding is also missing. The button at the bottom has text that is off center vertically. (same for inputs)

- [ ] Other than that, is it possible to change the text of the registration token error? It should probably be in dutch.

**Comment by JasperVandenberghen** [comment]
> * [x]  The image at the top is different/larger than the one in Figma. (You can export as png, jpg from Figma) Image should be around 300x125, is now 342×209 px
> * [x]  Spacing between some parts of the form is different from Figma (above 'Uw WebID', 'Uw Pod', the submit button). Also some missing margin at the bottom of the page. (white card sticks to bottom of page)
> * [x]  Input fields have a total height of 40px in Figma, included is vertical padding of +- 5px for the text. Horizontal padding is also missing. The button at the bottom has text that is off center vertically. (same for inputs)
> * [ ]  Other than that, is it possible to change the text of the registration token error? It should probably be in dutch.

Can't seem to retrieve the error messages from the node modules...

**Review by lem-onade — CHANGES_REQUESTED** [review]
- [ ] Still missing some margin at the bottom at the page:

  - [x] Remove `height: 100%` from `html`
image

  - [x] Change body margin to `40px auto`

- [x] The text inside the button at the bottom has a smaller font-size than Figma.
- [x] Inconsistent usage of 'je'/'jouw' and 'uw'. I suggest changing the titles to 'je'/'jouw'
- [ ] Color of text inside input fields should be dark blue, is now black
- [ ] Minor: the border around the input fields adds another 4px in height, which is why they're still a bit bigger than the submit button
- [ ] Minor: left-right padding on input fields is greater than top-bottom, from manage app: `padding: var(--gap-small) var(--gap-normal)` (small: 10px, normal: 20px);

**Review by lem-onade — DISMISSED** [review]

**Review by johannadm — CHANGES_REQUESTED** [review]
- border radius for buttons: no rounded corners 
- Image looks blurry, try using an image that is twice the size. (here: final image should be sized around 300x125px, so export a .png of 600x250px)

**Pull request merged**